orchestrator/packages/api/src/controllers/grading-queue-controller.ts

```diff
@@ -175,3 +177,18 @@ export const deleteJob = async (req: Request, res: Response) => {
     }
   }
 };
+
+export const getJobStatus = async (req: Request, res: Response) => {
```

Comment:
This should instead take in a list of keys, as each `Grade` is the product of the Submission and _one_ of its Graders, meaning that a single Submission can have multiple status strings.